database/rrdhost.c

```diff
@@ -798,7 +810,7 @@ struct label *load_config_labels()
         struct config_option *cv;
         for(cv = co->values; cv ; cv = cv->next) {
             char *name = cv->name;
-            if(is_valid_label_key(name) && strcmp(name, "from environment") && strcmp(name, "from kubernetes pods") ) {
+            if(is_valid_label_key(name) && is_valid_label_value(cv->value) && strcmp(name, "from environment") && strcmp(name, "from kubernetes pods") ) {
```

Comment:
shouldnt this be a function? like `is_valid_label_config_option`?

Reply:
I kept 

> > so they are working like primary key.
> 
> Label has name and value, right? There is no connection with `primary key` imo.

Yes, our documentation says this, the interpretation from my side happens based in the `libnetdata/config/appconfig.c`, I will rename to follow the documentation.

> > We cannot have two equal names
> 
> Is there the check in the code?

Yes we have, when we load configuration using `appconfig` every time that the labels are duplicated, Netdata overwrites the value, so we cannot have two labels with the same name when the label comes from `netdata.conf`, about the other sources(Kubernet and Docker), I cannot say anything.

Reply:
It could be, but we do not have a defined pattern for this, different collectors have `if` with various tests.
I liked your suggestion so I converted it to a `static inline` function.